_posts/network/2024-03-20-use-multiple-public-ips.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: page
+tags: [Network]
+page_title: Use multiple public ip's on one instance
+---
+
+This tutorial guides you through using multiple public ip addresses on an instance.
+Because in OpenStack there are multiple subnets, it's possible you get 2 ip addresses, in different subnets.
+This causes you to have multiple default gateways.
+
+## Introduction
```

Comment:
Is this also an issue on windows? if yes, please add the documentation for windows, if not, please make it clear in this chapter